core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp

```diff
@@ -326,7 +326,13 @@
                         "roleType": "SERVER"
                     }
                 ],
-                "serviceType": "ZOOKEEPER"
+                "serviceType": "ZOOKEEPER",
+                "serviceConfigs": [
+                    {
+                        "name": "service_config_suppression_server_count_validator",
+                        "value": "true"
```

Comment:
I think the warning should be suppressed only in the Light blueprint (`cdp-sdx.bp`).  Both HA blueprints have the recommended 3 ZK servers.